.github/actions/create-featurenet/action.yml

```diff
@@ -82,6 +86,14 @@ runs:
           echo "!!! Invalid feature net node image tag"
           exit 1
         fi
+        if [[
+          "${{ inputs.update }}" == "true" && \
+          ! "${{ inputs.rolling-update-partition }}" =~ ^[0-9]{1,2}$
```

Comment:
Technically we have only ten validators on Devnet, so range 0-9 is sufficient

Reply:
It's now changed to ^[0-9]$